dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataset/DataSet.java

```diff
@@ -552,6 +552,70 @@ public boolean isDataInputPeriodAndDateAllowed( Period period, Date date )
             .orElse( true );
     }
 
+    /**
+     * Returns a set of category options which are linked to this data set
+     * and also available for all organisation units.
+     */
+    public Set<CategoryOption> getAvailableCategoryOptions()
```

Comment:
Perhaps rename to something a bit more descriptive, like "getAssignedCategoryOptions()"